docs/src/community/maintainer.md

```diff
@@ -1,10 +1,10 @@
 # Current Maintainers
 
-| Name            | Role      | GitHub ID                                          |
-|-----------------|-----------|----------------------------------------------------|
-| Toru Komatsu    | Committer | [@utam0k](https://github.com/utam0k)               |
-| Thomas Schubart | Committer | [@Furisto](https://github.com/Furisto)             |
-| Yashodhan       | Committer | [@YJDoc2](https://github.com/YJDoc2)               |
-| Eric Fang       | Committer | [@yihuaf](https://github.com/yihuaf)               |
-| Sascha Grunert  | Committer | [@saschagrunert](https://github.com/saschagrunert) |
-| Jorge Prendes   | Committer | [@jprendes](https://github.com/jprendes)           |
+| Name            | Role      | GitHub ID                                          | Affiliation |
+|-----------------|-----------|----------------------------------------------------|-------------|
+| Toru Komatsu    | Committer | [@utam0k](https://github.com/utam0k)               | Independent |
+| Thomas Schubart | Committer | [@Furisto](https://github.com/Furisto)             | Independent |
+| Yashodhan       | Committer | [@YJDoc2](https://github.com/YJDoc2)               | Independent |
+| Eric Fang       | Committer | [@yihuaf](https://github.com/yihuaf)               | Independent |
+| Sascha Grunert  | Committer | [@saschagrunert](https://github.com/saschagrunert) | Red Hat     |
+| Jorge Prendes   | Committer | [@jprendes](https://github.com/jprendes)           | Docker      |
```

Comment:
My affiliation has changed and should now be Microsoft.

Reply:
@jprendes 
I didn't realize that. Updated. I'm looking forward to seeing you at KubeCon NA if possible.